src/components/Pages/WorkshopContent/WorkshopContent.js

```diff
@@ -0,0 +1,57 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+// import Markdown from 'markdown-to-jsx';
+
+import * as SC from './style';
```

Comment:
very clear what SC means

Reply:
[Link to the issue which explains the SC syntax and use of a style.js for each component](https://github.com/fac19/Zaatari-Radio/issues/2)